docs/content/tutorial/scheduling-your-pipeline.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: "Tutorial, part five: Scheduling your pipeline | Dagster Docs"
+description: Learn how to schedule your pipeline to run at a set frequency.
+---
+
+# Tutorial, part five: Scheduling your pipeline
+
+Now that you've written an entire pipeline in Dagster, you will need to run it regularly to keep your assets up to date.
+
+By the end of this part of the tutorial, you'll be able to:
+
+- Structure your project with code locations and jobs
+- Refresh your assets periodically with schedules
+
+---
+
+## Defining what assets to update
+
+A _job_ lets you target a selection of assets to materialize them together as a single action. Assets can belong to multiple jobs.
+
+Your Dagster repository has a file called `tutorial/__init__.py` that is used as a top-level definition for your project. Update the code in your to add the job using the <PyObject object="define_asset_job"/> function:
+
+```python file=/tutorial/scheduling/with_job/with_job.py
+from dagster import (
+    AssetSelection,
+    Definitions,
+    define_asset_job,
+    load_assets_from_modules,
+)
+
+from . import assets
+
+all_assets = load_assets_from_modules([assets])
+
+# Addition: define a job that will materialize the assets
+hackernews_job = define_asset_job("hackernews_job", selection=AssetSelection.all())
+
+defs = Definitions(
+    assets=all_assets,
+    jobs=[hackernews_job],  # Addition: add the job to Definitions object (see below)
+)
+```
+
+Dagster's <PyObject object="AssetSelection"/> module lets you choose which assets to attach to a job. In the example above, <PyObject object="AssetSelection" method="all"/> selects all assets.
+
+Once you have a job, you can execute it on a schedule, by clicking a button in the UI, the CLI, or via Dagster's GraphQL endpoints. Confirm that your job was defined by:
+
+1. Refreshing your deployment
+2. Going to the Dagster UI
+3. Navigating to **Overview > Jobs**
+4. Clicking on the job
+5. Seeing your assets selected for the job
+
+<Image
+alt="the Dagster UI with a job"
+src="/images/tutorial/job.png"
+width={1146}
+height={640}
+/>
+
+## About definitions
+
+Up until this point, you defined assets using the <PyObject object="asset" decorator /> decorator. Dagster definitions are entities that Dagster learns about by importing your code. Just now, you used a different kind of definition: a job definition.
+
+Managing one type of definition, such as assets, is easy. However, it can quickly become unruly as your project grows to have a variety of definitions (ex. schedules, jobs, sensors, etc.). Dagster provides a utility called the <PyObject object="Definitions" /> object. This <PyObject object="Definitions" /> object is used to combine all definitions into one.
```

Comment:
> Dagster provides a utility called the <PyObject object="Definitions" /> object.

🤔 By itself, this sentence feels a bit odd - I think it's because it feels abrupt and incomplete. What about combining this guy and the next sentence? Ex:

> To combine all definitions, Dagster provides a utility called the <PyObject object="Definitions" /> object.